pokemongo_bot/__init__.py

```diff
@@ -365,8 +365,11 @@ def item_inventory_count(self, id):
         self.api.get_player().get_inventory()
 
         inventory_req = self.api.call()
-        inventory_dict = inventory_req['responses'][
-            'GET_INVENTORY']['inventory_delta']['inventory_items']
+        try:
+            inventory_dict = inventory_req['responses'][
+                'GET_INVENTORY']['inventory_delta']['inventory_items']
+        except:TypeError:
```

Comment:
@TheSavior I got this error yesterday, it seems when call "api.call()" failed (may be due to the netwoking problems?) the "inventory_req" will get a NoneType, and thus the program will exit.